pkg/tss/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM linuxkit/alpine:43c139f87122e94af5fac9a9d1d96f0292ca1c0b AS build
+#FROM alpine:3.6 AS build
+
+RUN apk add --no-cache --initdb \
+    automake \
+    autoconf \
+    gettext \
+    gettext-dev \
+    git \
+    pkgconfig \
+    libtool \
+    libc-dev \
+    linux-headers \
+    gcc \
+    make \
+    openssl-dev \
+    util-linux \
+    && true
+
+RUN mkdir -p /usr/src
+COPY upload/src/glibc_stubs/ /usr/src/glibc_stubs
+WORKDIR /usr/src/glibc_stubs
+RUN make && make install
+
+RUN git clone https://git.code.sf.net/p/trousers/tpm-tools /usr/src/trousers-tpm-tools && cd /usr/src/trousers-tpm-tools && git checkout bdf9f1bc8f63cd6fc370c2deb58d03ac55079e84
+RUN git clone https://git.code.sf.net/p/trousers/trousers /usr/src/trousers-trousers && cd /usr/src/trousers-trousers && git checkout de57f069ef2297d6a6b3a0353e217a5a2f66e444
+WORKDIR /usr/src/trousers-trousers
+RUN sh bootstrap.sh && \
+    ./configure --prefix=/ --sysconfdir=/etc LDFLAGS="-L/out/lib/ -lgetpwent_r" && \
```

Comment:
It seems really weird to add a library you are using for compiling to `out/` as that is where you are installing. Just install the libraries it needs to build in `/usr/local/lib` and it will be much easier.

If necessary run `make install` twice, once to /usr/local and once to `/out` for things you need in the final image.